packages/@css-blocks/ember-app/src/brocolli-plugin.ts

```diff
@@ -63,9 +66,26 @@ export class CSSBlocksApplicationPlugin extends Filter {
     }
     let compiler = new BlockCompiler(postcss, config);
     let reservedClassnames = analyzer.reservedClassNames();
+    let testDataBuilder = new TestSupportDataGenerator();
     for (let block of blocksUsed) {
+
       let content: postcss.Result;
       let filename = importer.debugIdentifier(block.identifier, config);
+
+      if (!this.isProdApp) {
+        // Generate the test data
+        // Iterate over the blocks that belong to this app and its addons only for
+        // generating the test support data
+        if (block.identifier.startsWith(IDENTIFIER_PREFIX)) {
```

Comment:
There's a method to check for this in `BroccoliTreeImporter`:

```suggestion
        if (isBroccoliTreeIdentifier(block.identifier)) {
```